### PR TITLE
Remove all unnecessary binding in HTML

### PR DESCRIPTION
**Pull request by MarkvdVorst**
Closes issue [#396](https://github.com/wearefrank/ladybug-frontend/issues/396)

- Removed unnecessary bindings in html
- Added spaces behind or in front of double curly brackets ( {{ }} ) for consitancy with other code
- In test component, added a method to combine the strings instead of seperately adding them for better readability

### CODE REVIEW

**Review by Matthbo — DISMISSED** [review]

File: src/app/test/test.component.ts [hunk 1]
```diff
@@ -345,4 +345,8 @@ export class TestComponent implements OnInit, AfterViewInit {
   sortByName(): any[] {
     return this.reports.sort((a, b) => (a.name > b.name ? 1 : a.name === b.name ? 0 : -1));
   }
+
+  getFullPath(path: string, name: string): string {
+    return this.showRelativePath(path) + name;
```
Comment (MatthijsSmets) [inline src/app/test/test.component.ts:350]:
```suggestion
    return `${this.showRelativePath(path)}${name}`;
```

File: src/app/test/test.component.html [hunk 2]
```diff
@@ -68,27 +68,27 @@
               {{ report.storageId }}
             </td>
             <td>
-              <app-button data-cy-test="runReport" [icon]="'fa fa-play'" [title]="'Run report'" (click)="run(report.storageId)"></app-button>
-              <app-button data-cy-test="openReport" [icon]="'fa fa-folder-open'" [title]="'Open report'" (click)="openReport(report.storageId, report.name)"></app-button>
+              <app-button data-cy-test="runReport" icon="fa fa-play" title="Run report" (click)="run(report.storageId)"></app-button>
+              <app-button data-cy-test="openReport" icon="fa fa-folder-open" title="Open report" (click)="openReport(report.storageId, report.name)"></app-button>
             </td>
             <td>
-              {{ showRelativePath(report.path) }}{{ report.name }}
```
Comment (MatthijsSmets) [inline src/app/test/test.component.html:75]: can this be method be removed?

**Review by MatthijsSmets — APPROVED** [review]
Looks good, just two minor notes

File: src/app/shared/components/display-table/display-table.component.html [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
+      {{ report.waitingForStream ? 'Waiting for message to be' : 'Message is' }} captured asynchronously from
       a {{report.streaming.toLowerCase()}} stream
     </div>
 
     <div *ngIf="null === report.message" id="nullMessageLabel" class="btn btn-static  px-2">
       Message is null
     </div>
 
-    <div *ngIf="'' === report.message" id="emptyMessageLabel" class="btn btn-static  px-2">
+    <div *ngIf="!report.message" id="emptyMessageLabel" class="btn btn-static  px-2">
```
[the remark this answers is not in the document]
Comment (MarkvdVorst) [reply src/app/shared/components/display-table/display-table.component.html]: changed it back. there was a a second one where I did this as well which I changed back as well

File: src/app/test/test.component.ts [hunk 4]
```diff
@@ -345,4 +338,11 @@ export class TestComponent implements OnInit, AfterViewInit {
   sortByName(): any[] {
     return this.reports.sort((a, b) => (a.name > b.name ? 1 : a.name === b.name ? 0 : -1));
   }
+
+  getFullPath(path: string, name: string): string {
+    if (path) {
+      return `${path.replace(this.currentFilter, '')}/${name}`;
+    }
+    return `${path}/${name}`;
```
[the remark this answers is not in the document]
Comment (MarkvdVorst) [reply src/app/test/test.component.ts]: That should be '/${name}'. I added the change in the new commit

**Review by philipsens — APPROVED** [review]

**Review by MatthijsSmets — APPROVED** [review]

**Pull request merged**